### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by kl3sk**
ReadMe: Add more information about listener uses.

Add information from https://github.com/nucleos/NucleosDompdfBundle/pull/544

## Subject

Symfony listener and DomPDF hooks

### CODE REVIEW

**Comment by kl3sk** [comment]
@core23 Is it good for you ?

File: README.md [hunk 1]
```diff
@@ -139,6 +139,74 @@ The dompdf wrapper dispatches events to conveniently get the inner dompdf instan
...
+                    $size  = 48;
+                    $width = $fontMetrics->getTextWidth($text, $font, $size);
+                    $x     = ($canvas->get_width() - $width) / 2;
+                    $y     = ($canvas->get_height() - $size) / 2;
+                    $canvas->page_text($x, $y, $text, $font, $size, [0.5, 0.5, 0.5, "alpha" => 0.4], 20, 0, -45);
+                },
+            ],
+        ]
+    );
+    [...]
```
Comment (core23) [inline README.md]:
```suggestion
    $domPdf->setCallbacks([
        [
            'event' => 'begin_page_render',
            'f'     => function ($frame, $canvas, $fontMetrics) {
                // watermark
                $text  = 'WaterMak text';
                $font  = $fontMetrics->getFont('Helvetica', 'bold');
                $size  = 48;
                $width = $fontMetrics->getTextWidth($text, $font, $size);
                $x     = ($canvas->get_width() - $width) / 2;
                $y     = ($canvas->get_height() - $size) / 2;
                $canvas->page_text($x, $y, $text, $font, $size, [0.5, 0.5, 0.5, "alpha" => 0.4], 20, 0, -45);
            },
        ],
    ]);
```

File: README.md [hunk 2]
```diff
@@ -139,6 +139,74 @@ The dompdf wrapper dispatches events to conveniently get the inner dompdf instan
...
+```
+There are 6 events available, see documentation for more details:
```
Comment (core23) [inline README.md]: Please link to the document in the original DomPDF lib, because the number and the names might change.

Comment (kl3sk) [reply README.md]: I already link to it at the beginning. 

You may prefer at the end ?

File: README.md (hunk 1 above)
Comment (kl3sk) [reply README.md]: <s>My bad. I forgot to remove a level. </s>

I do it from my phone, it wasn't enough readable, it was just a copy/paste from documentation.
Indeed I prefer this notation too.

File: README.md (hunk 2 above)
Comment (kl3sk) [reply README.md]: I add an other link near the `events` names.

**Comment by core23** [comment]
Thanks @kl3sk

**Comment by kl3sk** [comment]
Hi, you're welcome

I have to re use your bundle in a new project, and don't see the updated readme in the landing page of the repo.

May I misunderstood something ?

**Pull request merged**